class.jetpack-gutenberg.php

```diff
@@ -89,6 +94,47 @@ class Jetpack_Gutenberg {
 	 */
 	private static $availability = array();
 
+	/**
+	 * Check to see if a block is able to run with current wp or gutenburg plugin versions
+	 *
+	 * @param array  $version_requirements An array containing wp and plugin version requirements.
+	 * @param string $extension_name The name of the block/plugin that has the version requirement.
+	 *
+	 * @since 8.4.0
+	 *
+	 * @return boolean True if the version of gutenberg required by the plugin is available.
+	 */
+	public static function is_editor_version_available( $version_requirements, $extension_name ) {
+		global $wp_version;
+
+		if ( defined( 'GUTENBERG_DEVELOPMENT_MODE' ) && GUTENBERG_DEVELOPMENT_MODE ) {
+			return true;
+		}
+
+		if ( defined( 'GUTENBERG_VERSION' ) ) {
+			$version_available = GUTENBERG_VERSION >= $version_requirements['plugin'];
```

Comment:
I wonder if we should check if `plugin` and `wp` are set before to call them here and below, to avoid any errors if someone registers a block but only specifies one or the other?

Reply:
Done - but wasn't sure on bailing if we just return false or true - open to suggestions on best approach here.